packages/one-app-dev-bundler/esbuild/plugins/generate-integrity-manifest.js

```diff
@@ -18,17 +18,18 @@ import ssri from 'ssri';
 import fs from 'node:fs';
 import { getJsFilenamesFromKeys } from '../utils/get-js-filenames-from-keys.js';
 
-async function writeIntegrityFragment(bundleName, integrityString, fileName) {
+// NOTE:: This needs to be synchronous to avoid race condition resulting in
+// missing integrity manifest entries
+function writeIntegrityFragment(bundleName, integrityString, fileName) {
   let integrityObject = {};
   try {
-    integrityObject = JSON.parse(await fs.promises.readFile(fileName, 'utf8'));
+    integrityObject = JSON.parse(fs.readFileSync(fileName, 'utf8'));
   } catch (e) {
     // empty catch in case file does not exist
   }
 
   integrityObject[bundleName] = integrityString;
-
-  return fs.promises.writeFile(fileName, JSON.stringify(integrityObject, null, 2));
+  return fs.writeFileSync(fileName, JSON.stringify(integrityObject, null, 2));
```

Comment:
I dont think we should be pretty formatting this, its just wasted bytes

Reply:
This file should be named in `.gitignore` and can be cleaned up between builds, I personally value the readability.

Reply:
Ah I see now. I was thinking this was the module config file.